src/shared/common/components/side-bar.jsx

```diff
@@ -40,14 +40,21 @@ class SideBar extends Component {
     }
 
     render() {
+        let peopleTabEnabled = true;
+        const { page } = this.props;
+        // last chars in caseId always correspond to case_type short code
+        // this check disabled People tab for WCS cases
+        if (page && page.params && page.params.caseId) {
+            peopleTabEnabled = !page.params.caseId.endsWith('c1');
```

Comment:
Are there any scenarios that we could have a case ID that ends with `c1` but isn't WCS, now or in the future? Is there a narrower thing we can match?